ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -243,3 +243,13 @@ SENSOR_CO2M_VOCM:
     - /SENSOR
     - CO2M
     - VOCM
+
+SENSOR_ZTM_ZHM_CO2M_SAQM:
```

Comment:
See comment above.

Reply:
updated. VOCM works too, thanks for the feedback.